mythril/laser/smt/bitvec_helper.py

```diff
@@ -3,10 +3,19 @@
 
 from mythril.laser.smt.bool import Bool, Or
 from mythril.laser.smt.bitvec import BitVec
+from mythril.laser.smt.array import BaseArray, Array
 
 Annotations = Set[Any]
 
 
+def _z3_array_converter(array: Union[z3.Array, z3.K]) -> Array:
```

Comment:
This is a helper for arrays, should it be in a file other than `bitvec_helper.py`?

Reply:
I should have looked at the comments I left before. This is  a duplicate